WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -944,20 +951,20 @@ - (NSString *)noResultsTitleText
 - (NSString *)noResultsMessageText
 {
     NSString *jetapackMessage   = NSLocalizedString(@"Jetpack supercharges your self-hosted WordPress site.", @"Notifications message displayed when a self-hosted user is not connected to Jetpack");
```

Comment:
I know you didn't change this as part of this PR, but small typo here - should be `jetpackMessage` :)